### ISSUE TITLE
Generated @available has trailing whitespace

### ISSUE THREAD

**Issue by niklassaers** [liamnichols/xcstrings-tool#97]
Hi,
I'm moving my project towards Swift 6, and Swift 6 is treating trailing space after @available as an error (before that it was a warning)

Example in my generated code:
```@available (*, deprecated, message: "Use `String.LocationAccess.notNowButton` instead. This property will be removed in the future.")```

This gives the error:
```Extraneous whitespace between attribute name and '('; this is an error in the Swift 6 language mode```

**Comment by liamnichols** [comment]
Thanks for reporting this! I didn't think much of it at first and TIL that it's an error in Swift 6!

I think that the issue actually comes from swift-syntax since I run the generated code through [it's formatter](https://github.com/apple/swift-syntax/tree/main/Sources/SwiftBasicFormat).

When I get a chance, I'll confirm and see if the issue has been addressed there. If we need to wait until September for a patch though, I'll figure out a workaround 👍

**Comment by niklassaers** [comment]
Thanks for explaining where it comes from. I pointed the build at the swift-syntax main branch instead of the pinned version and it seems to be resolved there. Here's the output from one of the tests included in xcstrings-tool:

```
     /// Creates a text view that displays a localized string defined in the ‘Localizable‘ strings table.
     public init(localizable: String.Localizable) {
−        if #available (macOS 13, iOS 16, tvOS 16, watchOS 9, *) {
+        if #available(macOS 13, iOS 16, tvOS 16, watchOS 9, *) {
             self.init(LocalizedStringResource(localizable: localizable))
             return
         }
```

**Comment by liamnichols** [comment]
Thanks for conforming this! Since SwiftSyntax 600.0.0 isn't going to be tagged and released until September (I assume), i'm going to push up a patch that works with older versions in the meantime.

**Comment by liamnichols** [comment]
Thanks for reporting this @niklassaers, it was resolved in 0.5.1 which has just been released 🙏

**Comment by niklassaers** [comment]
Awesome, thank you so much :-)
